js/content/common.js

```diff
@@ -1010,25 +1010,57 @@ let SteamId = (function(){
     };
 
     SteamId.prototype.getAccountId = function() {
-        return this._steamId.accountNumber * 2;
+        return this._steamId.accountNumber * 2 + this._steamId.y;
     };
 
     SteamId.prototype.getSteamId2 = function() {
-        return "STEAM_0:0:" + this._steamId.accountNumber;
+        return `STEAM_0:${this._steamId.y}:${this._steamId.accountNumber}`;
     };
     
 
     SteamId.prototype.getSteamId3 = function() {
-        return "[U:1:" + this.getAccountId() + "]";
+        const TYPES = { 0: "I", 1: "U", 2: "M", 3: "G", 4: "A", 5: "P", 6: "C", 7: "g", 10: "a" };
+        return `[${TYPES[this._steamId.type]}:1:${this.getAccountId()}]`;
     };
 
     SteamId.prototype.getSteamId64 = function() {
         return this._steamId64;
     };
 
+    SteamId.prototype.showDialog = function(path, name) {
+        function copySteamId(e) {
+            let elem = e.target.closest(".es_copy");
+            if (!elem) { return; }
+    
+            Clipboard.set(elem.innerText);
+    
+            let header = document.querySelector("#es_copy_header");
+            let headerText = header.innerText;
+            header.innerText = `${Localization.str.copied}`;
+            setTimeout(() => header.innerText = headerText, 1000);
```

Comment:
I still think that's not necessary

Reply:
I think otherwise. How else do you know the difference between a successful copy or a missclick.

Reply:
It's not like you have to do a pixel perfect click and the additional action of closing the window (that is useless now most likely) gets annoying